health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/BoundarySummary.js

```
@@ -0,0 +1,188 @@
+
+  const { isLoading, data, error, refetch } = Digit.Hooks.campaign.useSearchCampaign({
+    tenantId: tenantId,
+    filter: {
+      ids: [id],
+    },
+    config: {
+      select: (data) => {
+        const boundaryData = boundaryDataGrp(data?.[0]?.boundaries);
+        return {
+          cards: [
+            ...boundaryData?.map((item, index) => {
```

Biome

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)